packages/cloud-react/lib/overlay/Overlay/Background.tsx

```diff
@@ -0,0 +1,63 @@
+// Copyright 2023 @paritytech/polkadot-staking-dashboard authors & contributors
+// SPDX-License-Identifier: GPL-3.0-only
+
+import { ModalOverlay } from "../../base/modal/ModalOverlay";
+import { useAnimation } from "framer-motion";
+import { useEffect } from "react";
+import { useOverlay } from "../../hooks";
+import type { CanvasStatus } from "../OverlayProvider/types";
+
+export const Background = ({
+  externalOverlayStatus,
+}: {
+  externalOverlayStatus?: CanvasStatus;
+}) => {
+  const controls = useAnimation();
+  const {
+    modal: { status: modalStatus },
+    canvas: { status: canvasStatus },
+  } = useOverlay();
+
+  let { openOverlayInstances } = useOverlay();
```

Comment:
```suggestion
  const { openOverlayInstances } = useOverlay();
```

Reply:
If an external overlay is open, this has to increment, which is why `let` was opted for here.